packages/metrics/package.json

```diff
@@ -10,12 +10,15 @@
     "access": "public"
   },
   "scripts": {
-    "test": "npm run test:unit",
-    "test:unit": "jest --group=unit --detectOpenHandles --coverage --verbose",
-    "test:e2e:nodejs18x": "RUNTIME=nodejs18x jest --group=e2e",
-    "test:e2e:nodejs20x": "RUNTIME=nodejs20x jest --group=e2e",
-    "test:e2e:nodejs22x": "RUNTIME=nodejs22x jest --group=e2e",
-    "test:e2e": "jest --group=e2e",
+    "test": "vitest --run tests/unit",
+    "test:unit": "vitest --run tests/unit",
+    "test:unit:coverage": "vitest --run tests/unit --coverage.enabled --coverage.thresholds.100 --coverage.include='src/**'",
+    "test:unit:types": "echo 'Not Implemented'",
+    "test:unit:watch": "vitest tests/unit",
+    "test:e2e:nodejs18x": "RUNTIME=nodejs18x vitest --run tests/e2e",
+    "test:e2e:nodejs20x": "RUNTIME=nodejs20x vitest --run tests/e2e",
+    "test:e2e:nodejs22x": "RUNTIME=nodejs22x vitest --run tests/e2e",
+    "test:e2e": "vitest --run tests/e2e",
     "watch": "jest --group=unit --watch ",
```

Comment:
I initially accepted the suggestion but ended up removing it since vitest doesn't have a watch command. Watch mode is the default, and you just do a one-time run by passing the `--run` flag.